htdocs/compta/facture/class/facture.class.php

```diff
@@ -1730,14 +1730,17 @@ public function fetch($rowid, $ref = '', $ref_ext = '', $notused = '', $fetch_si
 					$this->error = $this->db->error();
 					return -3;
 				}
+
+				$this->db->free($resql);
+
 				return 1;
 			} else {
-				$this->error = 'Invoice with id='.$rowid.' or ref='.$ref.' or ref_ext='.$ref_ext.' not found';
-				dol_syslog(get_class($this)."::fetch Error ".$this->error, LOG_ERR);
```

Comment:
May be you can just replace with LOG_WARNING or LOG_DEBUG (In most cases, if we do a fetch, we are using an id and not founding the record is not the expected behaviour)

Reply:
Ok I replaced with LOG_WARNING in syslog when no record was found